README.md

```diff
@@ -33,11 +33,21 @@ You can download any icon of the pack directly from https://iconoir.com or get t
 ## Font
 
 Iconoir is available in .EOT, .SVG, .TTF, and .WOFF. Get them in the /fonts folder.
```

Comment:
I would move this paragraph to the end of the section because the basic usage is just: 1) Link to the style sheet 2) Use on HTML markup.

**Maybe rephrase also:**

The webfonts are in the [/fonts](./fonts/) folder. Iconoir is available in EOT, SVG, TTF, and WOFF formats.

Reply:
Does the updated version match your requests?